app/models/version.rb

```diff
@@ -53,6 +53,19 @@ class Version < ApplicationRecord # rubocop:disable Metrics/ClassLength
   validate :metadata_attribute_length
   validate :no_dashes_in_version_number, on: :create
 
+  scope :oldest_authored_at, lambda {
+    minimum(
+      <<~SQL.squish
+        CASE WHEN DATE(created_at) = '#{RUBYGEMS_IMPORT_DATE}'
+            AND built_at <= created_at THEN
+            built_at
+        ELSE
+            created_at
+        END
+      SQL
+    )&.in_time_zone
```

Comment:
The minimum query by itself returns a `Time` but `#authored_at` returns an `ActiveSupport::TimeWithZone`. So I added `in_time_zone` to convert the `Time` ➡️ `ActiveSupport::TimeWithZone`. 

But I don't think it is strictly necessary because the equality check in the tests pass without the conversion and the rendering on the versions page is the same as well.

Reply:
I think there is no timezone conversion currently happening in the codebase and all times are just printed to the page as stored in DB. IMHO not needed as well.

Reply:
makes sense, removed `in_time_zone`